assets/scss/components/_typeahead.scss

```diff
@@ -21,7 +21,17 @@
 	cursor: pointer;
 	padding: $space-half $space;
 
-	&--isActive {
-		background-color: $C_coolGrayLightTransp;
+	input[type="checkbox"],
+	input[type="radio"],
+	label {
+		pointer-events: none;
```

Comment:
This feels like a sloppy way to prevent default click behavior on inputs

Reply:
it's at least scoped to typeahead